src/models/contact.js

```diff
@@ -132,6 +132,39 @@ WebPage.attributes = {
   }),
 };
 
+class Groups extends RestfulModel {
+  toJSON() {
+    const json = {
+      id: this.id,
+      object: this.object,
+      account_id: this.accountId,
+      name: this.name,
+      path: this.path,
+    };
+    return json;
+  }
+}
+Groups.collectionName = 'groups';
+Groups.attributes = {
+  ...RestfulModel.attributes,
+  id: Attributes.String({
```

Comment:
Group will inherit `id`, `object` and `accountId` from the RestfulModel parent class, so I don't think we need to re-define them here

Reply:
ah, good point! I'll remove them